modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/security/SecurityScheme.java

```diff
@@ -46,33 +42,33 @@
   String description() default "";
 
   /**
-   * Required. The name of the header or query parameter to be used.
+   * Required. The name of the header or query parameter to be used. Applies to apiKey type.
    **/
-  String name();
+  String name() default "";
```

Comment:
I don't get it.  @arthurdm your PR is moving away from defaults but adding here? Why?